tutorials/07-fdem/plot_inv_1_em1dfm.py

```diff
@@ -271,7 +271,7 @@
 
 # Directive for the IRLS
 update_IRLS = directives.Update_IRLS(
-    max_irls_iterations=30, minGNiter=1, coolEpsFact=1.5, update_beta=True
+    max_irls_iterations=30, minGNiter=1, coolEpsFact=1.5
 )
```

Comment:
Shouldn't we need to make this example to use the new class? Otherwise I'd keep this example untouched (until we update it).